pages/common/logstash.md

```diff
@@ -0,0 +1,12 @@
+# logstash
+
+> Migrate data from one source into another.
```

Comment:
"Migrate" omits a major part of logstash, which is intermediate data processing/filtering. Also, the implication of move rather than copy is pretty important I think.